keyboards/ducky/one2mini/matrix.c

```diff
@@ -0,0 +1,229 @@
+/*
```

Comment:
Is the custom matrix necessary? 

I know for the ARM stuff, it was for a while. But there have been updates so that ARM boards don't need a custom matrix.

However, if the matrix is doing something "non standard", then a custom matrix may still be needed.  

Basically, we are trying to avoid using them, if possible.

Reply:
It's a small change - currently the matrix code switches the pins to InputHigh's instead of sending the level back to high on the output and it was messing with the Ducky in testing. My custom matrix.c simply changes that to a writePinHigh and leaves it as an output pin.
I don't have access to any other ARM boards, so I couldn't test to see if there was a reason for the output/input pin change in the original.

Reply:
Okay, that would be a good reason.